src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -1838,6 +1839,69 @@ static SimpleType[][] GetObjectwith2DArrayOfSimpleType()
         simpleType2D[1][1] = new SimpleType() { P1 = "1 1 value", P2 = 4 };
         return simpleType2D;
     }
+
+    [Fact]
+    public static void XmlSchemaTest()
+    {
+        XmlSchemas schemas = new XmlSchemas();
```

Comment:
Nit: use var when the type of the declaration is obvious from the initializer, especially if it is an object creation. This eliminates redundancy.